core/src/avm1/function.rs

```diff
@@ -472,26 +472,23 @@ impl<'gc> From<Gc<'gc, Avm1Function<'gc>>> for Executable<'gc> {
 /// Represents an `Object` that holds executable code.
 #[derive(Clone, Collect, Copy)]
 #[collect(no_drop)]
-pub struct FunctionObject<'gc> {
-    /// The script object base.
-    ///
-    /// TODO: Can we move the object's data into our own struct?
-    base: ScriptObject<'gc>,
-
-    data: GcCell<'gc, FunctionObjectData<'gc>>,
-}
+pub struct FunctionObject<'gc>(GcCell<'gc, FunctionObjectData<'gc>>);
 
 impl fmt::Debug for FunctionObject<'_> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.debug_struct("FunctionObject")
-            .field("ptr", &self.data.as_ptr())
+            .field("ptr", &self.0.as_ptr())
             .finish()
     }
 }
 
 #[derive(Clone, Collect)]
 #[collect(no_drop)]
 struct FunctionObjectData<'gc> {
+    /// The script object base.
+    ///
+    /// TODO: Can we move the object's data into our own struct?
```

Comment:
I think the TODO can be removed now

Reply:
I originally removed this TODO but then noticed that the `FunctionObjectData` has been present and in use since the beginning and just assumed that I misunderstood the comment :eyes:

Removing it... (unless someone sees another meaning behind it)